src/app/legacy/containers/ConsentBanner/index.amp.jsx

```diff
@@ -40,30 +41,35 @@ const AmpConsentWrapper = styled.div`
   }
 `;
 
-const Amp = () => (
-  <AmpConsentWrapper>
-    <AmpGeo />
-    <amp-consent id="consent" layout="nodisplay">
-      {jsonInlinedScript(ampConsentData)}
-      <div id="consent-prompt">
-        <Banner
-          type="privacy"
-          acceptAction="tap:cookie.show, privacy.hide, AMP.setState({ isManagingSettings: false }), dataCollectionHeading.focus"
-          rejectAction="tap:cookie.show, privacy.hide"
-          hideAction="tap:brandLink.focus, privacy.hide"
-          promptId="privacy"
-        />
-        <Banner
-          type="cookie"
-          acceptAction="tap:brandLink.focus, consent.accept"
-          rejectAction="tap:brandLink.focus, consent.reject"
-          hideAction="tap:brandLink.focus, cookie.hide"
-          promptId="cookie"
-          hidden
-        />
-      </div>
-    </amp-consent>
-  </AmpConsentWrapper>
-);
+const Amp = () => {
+  const { enabled } = useToggle('privacyPolicy');
+  return (
+    <AmpConsentWrapper>
+      <AmpGeo />
+      <amp-consent id="consent" layout="nodisplay">
+        {jsonInlinedScript(ampConsentData)}
+        <div id="consent-prompt">
+          {enabled && (
+            <Banner
+              type="privacy"
+              acceptAction="tap:cookie.show, privacy.hide, AMP.setState({ isManagingSettings: false }), dataCollectionHeading.focus"
```

Comment:
Think there might be an existing UX bug for isManagingSettings. This seems to just be used for style targeting https://github.com/bbc/simorgh/blob/211c6cce462803c0e1cf2a25dc3db5a17e368392/src/app/legacy/containers/ConsentBanner/Banner/cookie.amp.jsx#L160. However, there's nothing that changes this style, I think it stopped working as an extra item to on was added https://github.com/bbc/simorgh/blob/211c6cce462803c0e1cf2a25dc3db5a17e368392/src/app/legacy/containers/ConsentBanner/Banner/cookie.amp.jsx#L293